vars/mavenExecute.groovy

```diff
@@ -63,12 +65,9 @@ void call(Map parameters = [:]) {
 
         String command = "mvn"
 
-        def globalSettingsFile = configuration.globalSettingsFile
-        if (globalSettingsFile?.trim()) {
-            if(globalSettingsFile.trim().startsWith("http")){
-                downloadSettingsFromUrl(globalSettingsFile)
-                globalSettingsFile = "settings.xml"
-            }
+        def globalSettingsFile = configuration.globalSettingsFile?.trim()
+        if (globalSettingsFile) {
+            globalSettingsFile = new Utils().downloadMavenSettingsFromUrlIfRequired(this, globalSettingsFile as String)
```

Comment:
Isn't it possible to import the specific static method and then use it without instantiating with `new`?

Reply:
Yes, updated.